docs/manual/mod/mod_setenvif.xml

```diff
@@ -59,6 +59,16 @@ BrowserMatch MSIE !netscape
    are not separately evaluated in the subrequest due to the API phases
    <module>mod_setenvif</module> takes action in.</p>
 
+   <p>A regular expression only needs quoting when it contains space, 
+   in which case single and double quotes are equivalent. Unlike 
+   <code class="module"><a 
```

Comment:
Hey, @nisbet-hubbard , I really like this PR, but have a couple of nitpicks.

Please use <module> (see line 60 above for example) rather than <code class="module">